financialaid/models.py

```diff
@@ -0,0 +1,110 @@
+"""
+Models for the Financial Aid App
+"""
+
+from django.contrib.auth.models import User
+from django.db import (
+    models,
+    transaction,
+)
+from jsonfield import JSONField
+
+from courses.models import (
+    CourseRun,
+    Program,
+)
+
+
+class Tiers(models.Model):
+    """
+    The possible tiers to be used
+    """
+    name = models.TextField()
+    description = models.TextField()
+
+
+class TierProgram(models.Model):
+    """
+    The tiers for discounted pricing assigned to a program
+    """
+    program = models.ForeignKey(Program, null=False)
+    tier = models.ForeignKey(Tiers, null=False)
+    value = models.IntegerField(null=False)
+    current = models.BooleanField(null=False, default=False)
+
```

Comment:
I am confused by what you are saying, but I think this change can be made when we implement the algorithm.